sites/default/sqlconf.php

```diff
@@ -1,16 +1,17 @@
 <?php
 //  OpenEMR
 //  MySQL Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'openemr';
-$pass	= 'openemr';
-$dbase	= 'openemr';
+$pass	= '123456789';
```

Comment:
Hello @MatthewVita 
Sorry for making check in the configuration file
Let us know what need to do make it effective.

Thanks
Rishabh Software